hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/fs/ozone/TestLeaseRecovery.java

```diff
@@ -149,8 +149,7 @@ public void testRecovery() throws Exception {
         Thread.sleep(1000);
       }
       // The lease should have been recovered.
-      assertTrue("File should be closed", fs.recoverLease(file));
```

Comment:
Yes, the second call of recoverLease will fail if the first has succeeded, since the file is already committed so checks in OM side will fail.  Should we keep the same behavior as HDFS? I remember @szetszwo  has mentioned a case that if after a Hbase region server fails, two new Hbase region servers start on the server, if the second region server calls recoverLease and gets a successful result, then there could be two region servers started and running on the same server.  Not sure if Hbase has other checks to prevent the second region server to start so that the two region servers running altogether will not happen.